spec/integration/alaveteli_pro/view_request_spec.rb

```diff
@@ -88,6 +88,26 @@
     end
   end
 
+  context 'the user does not have an active subscription' do
+
+    before do
+      pro_user.pro_account.update!(stripe_customer_id: nil)
```

Comment:
As mentioned above, this is not what happens. The removal of the role is what we care about here.

Reply:
More importantly, I think this should contain a spec to assert that it _doesn't_ allow a user to extend an embargo.